__std__/cells/plutus/library/plutus-project.nix

```diff
@@ -190,7 +190,7 @@ let
   });
 
 in
-
-project.appendOverlays [
+{ compiler-nix-name ? library.ghc-compiler-nix-name }:
```

Comment:
if you move this to line 2, then `project` on line 20 does not need the `{ compiler-nix-name }:`